apps/foundation-demo/src/app/cart/components/cart-async-wrapper/cart-async-wrapper.component.scss

```diff
@@ -26,10 +26,11 @@
   }
 
   &__loading-icon {
-    margin: 20px auto;
+    padding: 40px calc(50% - 37px);
```

Comment:
Is there a way for us to refactor this so that the code is housed in one place across all of these components? See #55

Reply:
This is out of scope of this PR.